css/styles.css

```diff
@@ -0,0 +1,16 @@
+@media (max-width: 75em) {
+  html {
+    font-size: 60%;
+  }
+}
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background: var(--main-red);
+  box-shadow: 0 2px 0 rgba(0, 0, 0, 0.4);
+  z-index: 10;
+}
```

Comment:
after you finish typing your code, It will be more interesting to leave another blank line (only 1 single line) it's a better way to code 😃  right ?
It will avoid you to get errors from linters verification in the future

Reply:
Well done for describing the necessary changes and the reasons they are needed👍, but I think you can tone done your suggestion so that it won't sound bossy to the student.☺
e.g ' I think after you finish typing your code...'